src/services/root.js

```diff
@@ -0,0 +1,54 @@
+const { createTimestampHook } = require('@albert-team/mongol/builtins/hooks')
+const nanoid = require('nanoid/async')
+
+class RootController {
+  constructor(mongol) {
+    this.userCollection = mongol.database.collection('users')
+    mongol.attachDatabaseHook(this.userCollection, createTimestampHook())
+  }
+
+  getAllUsers() {
+    return this.userCollection.find({}).toArray()
+  }
+
+  createUser(username, name, avatar, email, birthday, linkedAccounts) {
```

Comment:
```suggestion
  createUser(data) {
  const { username, name, avatar, email, birthday, linkedAccounts } = data
```
Ko nên để hàm có nhiều tham số thế